public/graph.js

```diff
@@ -16,6 +16,7 @@ class GraphController {
         let isYLimitReceivedManuallySet = false,
             isYLimitSentManuallySet = false,
             dayDateFormat = d3.timeFormat("%Y-%m-%d"),
```

Comment:
Lets format this and use it instead of creating a new variable

Reply:
` data.forEach(function(d) {
            d.datetime = new Date(d.datetime);
            d.day = dayDateFormat(new Date(d.datetime));`
due to this if we format the first variable we'll get the error below
![err](https://user-images.githubusercontent.com/39700595/74811207-0f4d8500-5302-11ea-9663-0ccbc7c0945b.png)

Reply:
Ahh i see thanks.